### PR TITLE
Update Go versions and vendor using Godep (and fix the build)

### PR DESCRIPTION
**Pull request by DanielNi**
- Vendor latest versions of dependencies using Godep.
- Update supported Go versions and move to the native "context" package
- Regenerates the language using the latest version of peg
- Don't check licenses in vendor/
- Workaround to fix a Cassandra startup bug in Travis CI

All of this to get the build to pass...

### CODE REVIEW

File: query/command/command.go [hunk 1]
```diff
@@ -15,6 +15,7 @@
 package command
 
 import (
+	netcontext "context"
```
Comment (drcapulet) [inline query/command/command.go]: I think we could probably remove the `netcontext` here?

**Comment by DanielNi** [comment]
@drcapulet I guess we could import the peg library somewhere (maybe in `parser.go`?), otherwise I don't think it'll work with Godep.

**Comment by bwester** [comment]
Godep will only vendor imported packages, not commands. I wouldn't want to roll something on our own here just to get a stable copy of `peg`. Checking in generated source code will work well enough.

That Travis CI build issue you mentioned is from 2015. How has this project not run into this problem before?

**Comment by DanielNi** [comment]
Looking at previous builds, it looks like Cassandra has always failed to start up, but the tests still passed somehow. Maybe something in the newer versions of Go causes the tests to fail.

**Pull request merged**